src/user.js

```diff
@@ -5,6 +5,7 @@ class User {
     this.pantry = user.pantry;
     this.favoriteRecipes = [];
     this.recipesToCook = [];
+    this.recipesCooked = [];
```

Comment:
Let's talk about these new properties and what they mean a little more next time we meet up. I think we're heading in a good direction, but I think we just need to iron out exactly what our goal is for adding/deleting ingredients and how these properties will come into play.

Reply:
Yeah, will be good to go over these.